src/parser/getDepositData.spec.ts

```diff
@@ -15,12 +15,12 @@ describe('getDepositData',() => {
   const networks: SupportedNetworks[] = [ 'holesky', 'mainnet', 'gnosis', 'chiado' ]
 
   networks.forEach(network => {
-    it(`processes valid amount with "${network}" network`, () => {
+    it(`processes valid amount with "${network}" network`, async () => {
       const data: DepositDataInput = { ...mockInput, network }
       const amount = getAmount(network)
-      const result = getDepositData(data)
+      const result = await getDepositData(data)
 
-      expect(getDepositData(data)).toEqual(result)
+      expect(await getDepositData(data)).toEqual(result)
```

Comment:
I think we should mock result and compare it with the getDepositData call instead of comparing two getDepositData calls

Reply:
https://github.com/stakewise/v3-deposit-data-parser/pull/9/commits/e53eca070b00b01f6197c8ed5d92dcc2f202774a#diff-80cf3d2a1152c6e66e8dea8091050df9a5d1f4e638de5c0df037b336ba33f943L23